packages/frontend/src/components/Markdown.tsx

```diff
@@ -15,39 +15,15 @@ const markdown = MarkdownIt({
   typographer: true,
 }).use(outLinksPlugin)
 
-function outLinksPlugin(md: MarkdownIt) {
-  const defaultRender =
-    md.renderer.rules.link_open ??
-    function (tokens, idx, options, env, self) {
-      return self.renderToken(tokens, idx, options)
-    }
-
-  md.renderer.rules.link_open = (tokens, index, options, env, self) => {
-    const token = tokens[index]
-    if (isOutLink(token.attrGet('href'))) {
-      token.attrSet('target', '_blank')
-      token.attrSet('rel', 'noreferrer noopener')
-    }
-    return defaultRender(tokens, index, options, env, self)
-  }
-}
-
 export function Markdown(props: MarkdownProps) {
-  if (props.inline) {
-    const rendered = markdown.renderInline(props.children)
-    return (
-      <span
-        className={cn('mdc', props.className)}
-        dangerouslySetInnerHTML={{ __html: rendered }}
-      />
-    )
-  } else {
-    const rendered = markdown.render(props.children)
-    return (
-      <div
-        className={cn('mdc', props.className)}
-        dangerouslySetInnerHTML={{ __html: rendered }}
-      />
-    )
-  }
+  const rendered = props.inline
+    ? markdown.renderInline(props.children)
+    : markdown.render(props.children)
+
+  return (
+    <div
```

Comment:
if inline it should be span.
Maybe you could say `const Element = props.inline ? 'span' : 'div'`